can_tools/scrapers/official/TN/tn_state.py

```diff
@@ -0,0 +1,208 @@
+import pandas as pd
+import requests
+import us
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import CountyDashboard
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class TennesseeCasesDeathsTests(CMU, StateDashboard):
+    """
+    Fetch state level covid data from official state of Tennessee spreadsheet
+    """
+
+    source = (
+        "https://www.tn.gov/content/tn/health/cedep/ncov/data.html"
+        "https://www.tn.gov/health/cedep/ncov/data/downloadable-datasets.html"
+    )
+    state_fips = int(us.states.lookup("Tennessee").fips)
+    has_location = True
+    location_type = "state"
+
+    def fetch(self) -> requests.Response:
+        # Set url of downloadable dataset
+        url = (
+            "https://www.tn.gov/content/dam/tn/health/documents/cedep/"
+            "novel-coronavirus/datasets/Public-Dataset-Daily-Case-Info.XLSX"
+        )
+        request = requests.get(url)
+        return request.content
+
+    def normalize(self, data) -> pd.DataFrame:
+        # Read data into data frame
+        df = pd.read_excel(data, parse_dates=["DATE"])
+
+        # Rename columns
+        df = df.rename(columns={"DATE": "dt"})
+
+        # Create dictionary for columns to map
+        crename = {
+            "TOTAL_CONFIRMED": CMU(
+                category="cases", measurement="cumulative", unit="people"
+            ),
+            "TOTAL_PROBABLE": CMU(category="cases", measurement="new", unit="people"),
```

Comment:
I believe the interpretation of new is "was reported since yesterday"

If so, I don't think that this is categorized correctly based on name `TOTAL_PROBABLE`

Also FWIW I believe we decided to stop tracking probable cases because of how differently this is reported across states or counties.

@cc7768 please fact check me here

Reply:
You are correct, I think I was using the old columns and finding their best match before.  I think a more relevant column would be the "NEW_CONFIRMED" column.  I'll update.